util.go

```diff
@@ -240,15 +242,21 @@ func asIntFromTimeDuration(param string) int64 {
 	return int64(d)
 }
 
+// GlobalTimeDurationTypes holds specific application registered custom
+// time.Duration types so their values can be properly parsed as time.Duration.
+var GlobalTimeDurationTypes = []reflect.Type{timeDurationType}
```

Comment:
AFACT no need for this to be exported:
```suggestion
var globalTimeDurationTypes = []reflect.Type{timeDurationType}
```